src/constructs/proxyServer/integ.proxyServer.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This file can be removed from here and inserted from integ test when need inorder to produce a .js file for testing